analysis/webservice/algorithms/doms/domsconfig.ini

```diff
@@ -1,5 +1,5 @@
 [cassandra]
-host=127.0.0.1
+host=sdap-cassandra
```

Comment:
I understand why you did it but this change along with removing the `docker-entrypoint.sh` file essentially means that the hostname for the solr and cassandra services are now hard-coded into the docker image.

To be fair, this is actually a problem with how the application itself loads the ini config files and needs to be fixed outside of this PR.

Reply:
I agree, this should be addressed in a separate PR... unfortunately without `docker-entrypoint.sh` these values are indeed hardcoded, but I had to remove docker-entrypoint because the Spark Operator controller now handles the submission of the of the webapp to the spark server. I think addressing this should be a top priority after this PR goes in